azure/services/backoff.go

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
I thought the retry handling was going to wait until a followup PR? Or is this just going to be the first place where implemented?

Reply:
+1 let's remove this file

Reply:
done